docker/allensdk/Dockerfile

```diff
@@ -1,11 +1,29 @@
-# DOCKER-VERSION 1.11.1
+# DOCKER-VERSION 2.2.0.5
 #
 # docker build --tag alleninstitute/allensdk .
 # docker run -it alleninstitute/allensdk /bin/bash
 #
-FROM alleninstitute/anaconda_neuron_1370
+FROM continuumio/anaconda3
 
-MAINTAINER Tim Fliss <timf@alleninstitute.org>
+WORKDIR /root
 
-COPY . ./allensdk/
-RUN  pip install ./allensdk
\ No newline at end of file
+# Add AllenSDK and NEURON libraries
+RUN pip install -q allensdk neuron
+
+# Install NEURON
+RUN wget https://neuron.yale.edu/ftp/neuron/versions/v7.7/nrn-7.7.x86_64-linux.deb; \
+	dpkg -i nrn-7.7.x86_64-linux.deb; \
+	apt-get install -yq \
```

Comment:
Like above the q option might make it more difficult to debug in the future if anything ever breaks.

Reply:
Should be fixed.